cupy/cuda/pinned_memory.pyx

```diff
@@ -321,6 +319,6 @@ cdef class PinnedMemoryPool:
             int: The total number of free blocks.
         """
         cdef Py_ssize_t n = 0
-        for v in six.itervalues(self._free):
+        for v in self._free.itervalues():
```

Comment:
Let me take a note here. Actually, we got an error for `self._free.itervalues()` with python3. Fixed via https://github.com/cupy/cupy/pull/295